### PR TITLE
Added two test files

### PR DESCRIPTION
**Pull request by aditya1702**
Added tests for the two view functions added in PR #236, returning a JSON response

### CODE REVIEW

File: test_get_session_by_id.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+import unittest
+
+from tests.setup_database import Setup
+from tests.object_mother import ObjectMother
+from open_event import current_app as app
+from open_event.helpers.data import save_to_db
+
+
+class Test_Get_Session_By_Id(unittest.TestCase):
```
Comment (juslee) [inline test_get_session_by_id.py]: Codacy Issue found: [Invalid class name "Test_Get_Session_By_Id"](https://www.codacy.com/app/triplez/open-event-orga-server/file/2499325612/issues/source?bid=3233864&fileBranchId=3271518#l10)

File: test_get_speaker_by_id.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import unittest
+from tests.setup_database import Setup
+from tests.object_mother import ObjectMother
+from open_event import current_app as app
+from open_event.helpers.data import save_to_db
+
+
+class Test_Get_Speaker_By_Id(unittest.TestCase):
```
Comment (juslee) [inline test_get_speaker_by_id.py]: Codacy Issue found: [Invalid class name "Test_Get_Speaker_By_Id"](https://www.codacy.com/app/triplez/open-event-orga-server/file/2499325613/issues/source?bid=3233864&fileBranchId=3271518#l8)

**Comment by aditya1702** [comment]
@rafalkowalski @leto added the tests. But why is the Travis CI failing?

**Comment by leto** [comment]
You have a test called "test_get_session_by_id(self)" in both files, probably.

Also, both tests have exactly the same setup and teardown methods, as many tests will. It would be much better if you created a TestCase class that both inherit from. Then there will be a lot less duplication as we write more tests.

**Comment by aditya1702** [comment]
Ok @leto will update and make a fresh PR :)

**Pull request closed without merging**